google/cloud/firestore_v1/document.py

```diff
@@ -26,7 +26,7 @@
 from google.cloud.firestore_v1.types import write
 from google.cloud.firestore_v1.watch import Watch
 from google.protobuf import timestamp_pb2
-from typing import Any, Generator
+from typing import Any, Callable, Generator, Iterable
```

Comment:
I guess `Callable` and `Iterable` are being used in code from the other commit?

Reply:
Yes.